roles/katello_repositories/tasks/staging_repos.yml

```diff
@@ -40,6 +40,16 @@
   when:
     - katello_repositories_pulp_version == "nightly"
 
+- name: 'Add Pulpcore repository'
+  yum_repository:
+    name: pulpcore-repository
+    description: Pulpcore nightly repository
+    baseurl: "https://fedorapeople.org/groups/katello/releases/yum/nightly/pulpcore/el{{ ansible_distribution_major_version }}/x86_64/"
```

Comment:
This is the release repo, not staging. Take a look at the above koji repos.

Reply:
@ekohl got it, thanks